tools/document-pii/src/main.rs

```diff
@@ -0,0 +1,267 @@
+            continue;
+        }
+        if entry.file_type().is_file() && entry.path().extension().map_or(false, |ext| ext == "rs")
+        {
+            rs_files.push(entry.into_path());
+        }
+    }
+    rs_files
+}
+
+/// Prints documentation for metrics.
+#[derive(Debug, Parser, Default)]
+#[command(verbatim_doc_comment)]
+pub struct Cli {
+    /// Optional output path. By default, documentation is printed on stdout.
+    #[arg(short, long)]
+    pub output: Option<PathBuf>,
+
+    /// Path to the rust crate/workspace
+    #[arg(short, long)]
+    pub path: Option<PathBuf>,
+
+    /// The struct or enum of which you want to find all pii fields. Checks all items if none is
+    /// provided.
+    #[arg(short, long)]
+    pub item: Option<String>,
+
+    /// Vector of which pii-values should be looked for, options are: "true, maybe, false".
+    #[arg(long, default_value = "true")]
+    pub pii_values: Vec<String>,
+}
+
+impl Cli {
+    fn write_pii<W: Write>(&self, mut writer: W, metrics: &[String]) -> anyhow::Result<()> {
+        for metric in metrics {
+            writeln!(writer, "{}", metric)?;
+        }
+
+        Ok(())
+    }
+
+    pub fn run(self) -> anyhow::Result<()> {
+        // User must either provide the path to a rust crate/workspace or be in one when calling this script.
+        let path = match self.path.clone() {
+            Some(path) => path,
+            None => std::env::current_dir()?,
+        };
+
+        if !path.join("Cargo.toml").exists() {
+            panic!("Please provide the path to a rust crate/workspace");
+        }
+
+        let rust_file_paths = find_rs_files(&path);
+
+        // Before we can iterate over the pii fields properly, we make a mapping between all
+        // paths to types and their AST node, and of all modules and the items in their scope.
+        let TypesAndScopedPaths {
+            all_types,
+            scoped_paths: use_statements,
+        } = AstItemCollector::get_types_and_scoped_paths(&rust_file_paths)?;
+
+        let pii_types = match self.item.as_deref() {
+            // If user provides path to an item, find pii_fields under this item in particular.
+            Some(path) => {
+                find_pii_fields_of_type(path, &all_types, &use_statements, &self.pii_values)
+            }
+            // If no item is provided, find pii fields of all types in crate/workspace.
+            None => find_pii_fields_of_all_types(&all_types, &use_statements, &self.pii_values),
+        }?;
+
+        // Function also takes a string to replace unnamed fields, for now we just remove them.
+        let output_vec = get_pii_fields_output(pii_types, "".to_string());
+
+        match self.output {
+            Some(ref path) => self.write_pii(File::create(path)?, &output_vec)?,
+            None => self.write_pii(std::io::stdout(), &output_vec)?,
+        }
+
+        Ok(())
+    }
+}
+
+/// Represent the pii fields in a format that will be used in the final output.
```

Comment:
In some cases you use PII in some pii, let's use PII everywhere in the docs, please.  There are many places like this. 

```suggestion
/// Represents the PII fields in a format that will be used in the final output.
```